maths/Ugly_numbers.dart

```diff
@@ -0,0 +1,37 @@
+int maxDivide(int a, int b) {
```

Comment:
You could add some tests
```suggestion
import 'package:test/test.dart';

int maxDivide(int a, int b) {
```